src/common/abstractResume.ts

```diff
@@ -18,6 +17,8 @@ import {
 import { jobId, requiredDoceOrgFlag, useMostRecent, wait } from '../common/flags';
 import DoceMonitor from '../streamer/doceMonitor';
 import { DeployPipelineCache } from './deployPipelineCache';
+import { OutputServiceFactory } from './outputService/outputServiceFactory';
+import { ResumeCommandOutputService } from './outputService/resumeCommandOutputService';
```

Comment:
Maybe we can add an Index.ts for output, so we just need to import all from ./outputService only ???

Reply:
You mean creating an index.ts file in the outputService directory and adding there just the imports for everything?